### PR TITLE
Remove GCP and GKE specific info from readme description

### PR DESCRIPTION
**Pull request by eduartua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 # Terraform Modules
 
-This repo contains [Terraform](https://www.terraform.io) modules for running a Private Kubernetes cluster on [Google Cloud Platform (GCP)](https://cloud.google.com/)
-using [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/).
+This repo contains [Terraform](https://www.terraform.io) modules for running a Private Kubernetes cluster.
```
Comment (victortrac) [inline README.md]: Why does this have to be limited to a private kubernetes cluster?

Comment (eduartua) [reply README.md]: As we can add any module, there is no reason for that. Fixing it now.

**Comment by victortrac** [comment]
@eduartua Why is the GKE service account its own separate module?  Why not just include it in the main GKE module?

**Comment by eduartua** [comment]
> @eduartua Why is the GKE service account its own separate module? Why not just include it in the main GKE module?

First, I thought about building a service accounts module. I was to add everything about service accounts in that module, but with the velero service account module I realized that it is easier to have them separated. But we can move that to the GKE module.

**Pull request merged**